sensor_msgs/Image RGB_im
---
float32[] data
